languages/da/Zend_Captcha.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright  Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license    http://framework.zend.com/license/new-bsd     New BSD License
+ */
+
+/**
+ * EN-Revision: 30.Jul.2011
```

Comment:
date we modified iirc, see 'id' translation: https://github.com/zendframework/zend-i18n-resources/blob/master/languages/id/Zend_Captcha.php#L11